### ISSUE TITLE
Opera

### ISSUE THREAD

**Issue by saiichihashimoto** [kogg/hovercards#4]
- [x] HoverCards works in opera
- [x] Automatically builds and deploys to the opera webstore
  - [This is not possible](https://twitter.com/shwetank/status/763416103242653696)
- [ ] HoverCards is listed in the opera webstore
- [ ] Inline install from hovercards.com

https://dev.opera.com/extensions/basics/
https://dev.opera.com/extensions/apis/
https://dev.opera.com/extensions/inline-installation/

**Comment by shwetank** [comment]
Hi,

I'm PM of extensions for Opera. Hovercards is great, and it works great in Opera (I've sideloaded it and it works, since opera is based on the same chromium extensions model as Chrome). 

Please do submit to addons.opera.com - I'll be happy to feature it as a recommended extension on the homepage too.

**Comment by saiichihashimoto** [comment]
Hey Shwetank, that would be great! One hesitation is that, unlike chrome, I wouldn't be able to have Travis [automatically deploy to the opera addon store](https://twitter.com/shwetank/status/763416103242653696) for me. I would have to manually upload each version. I would like to keep continuous deployment going but I'm not sure how I would properly do that with opera.

**Comment by shwetank** [comment]
Yes, we don't have that capability yet. So you'll have to manually submit changes. How often do you submit updates?

**Comment by saiichihashimoto** [comment]
Currently, not often. We're not actively pursuing work on HoverCards, though it's open for collaboration and we will occasionally work on it. A chunk of our users are already using HoverCards in Opera so this might be worth doing sometime soon.

**Comment by shwetank** [comment]
If its not going to be updated all too often, then maybe its worth to upload it manually? The process to do it is pretty straighforward and I can help out in case you need any help :)

**Comment by saiichihashimoto** [comment]
Yeah, I had it working for myself at one point as well. Currently, I don't have a lot of time to be working on HoverCards. This is one to come back to.
